guides/15_Background_Filter_Video_Processor.md

```diff
@@ -74,48 +74,73 @@ Cross-Origin-Embedder-Policy: require-corp
 
 when serving your application HTML. These headers are optional but advised: they improve web security, as well as allowing the browser to support technologies like `SharedArrayBuffer`.
 
-## Checking for support before offering background blur
+## Checking for support before offering background filtering
 
-Some browsers support the Amazon Chime SDK but do not support background blur. Additionally, some devices are not fast enough to keep up with real-time video while blurring the background.
+Some browsers support the Amazon Chime SDK but do not support background filtering. Additionally, some devices are not fast enough to keep up with real-time video while filtering the background.
 
 The SDK provides an async static method to allow you to cheaply check for the required browser features:
 
 ```typescript
+// Background Blur
 import { BackgroundBlurVideoFrameProcessor } from 'amazon-chime-sdk-js';
 await BackgroundBlurVideoFrameProcessor.isSupported();
+
+// Background Replacement
+import { BackgroundReplacemenetVideoFrameProcessor } from 'amazon-chime-sdk-js';
+await BackgroundReplacementVideoFrameProcessor.isSupported();
 ```
 
 If `isSupported` resolves to `true`, you can instantiate the blur video frame processor.
 
 ```typescript
+// Background Blur
 const processors = [];
 if (await BackgroundBlurVideoFrameProcessor.isSupported()) {
     const blurProcessor = await BackgroundBlurVideoFrameProcessor.create();
     processors.push(blurProcessor);
 }
-```
 
-The create method will pre-load the model file and prepare background blur for use. Failure to load necessary resources will result in `isSupported` returning `false`. In the event that the builder creates the processor and it is not supported a no-op video frame processor will be returned and background blur will not be applied.
+// Background Replacement
+const processors = [];
```

Comment:
nit: You cannot create two `const` variables. Remove this line, or put the background replacemenet into another code block.

Reply:
fixed